drivers/scheduler/k8s/specs/elasticsearch/csi/px-storage-class.yaml

```diff
@@ -0,0 +1,8 @@
+kind: StorageClass
```

Comment:
This shouldn't be csi. csi is the provisioner, not the driver. You don't need a separate storage class for this. Previously if driver was pxd and provisioner was csi it would just change the povisioner from in-tree to csi.

Also, we'll be adding specs for csi with other cloud providers, so you can't just add this as the csi driver

Reply:
Yes we should just stick to replacing the provisioner in storage class as it's being done currently.
@harsh-px thoughts?